alacritty/src/display/mod.rs

```diff
@@ -490,6 +490,7 @@ impl Display {
         for cell in &mut content {
             grid_cells.push(cell);
         }
+
```

Comment:
I kinda like it without this newline, since it keeps all the `content.` stuff together. Not that big of a deal though.